torchaudio/datasets/utils.py

```diff
@@ -171,6 +172,38 @@ def download_url(
             )
 
 
+def download_url(urls, *args, max_workers=5, **kwargs):
+    """Download urls to disk. The other arguments are passed to download_single_url.
+
+    Args:
+        urls (str or List[str]): List of urls.
+        max_workers (int): Maximum number of workers.
+    """
+
+    if isinstance(urls, str):
+        return download_single_url(urls, *args, **kwargs)
+
+    # Turn arguments into lists
+    args = list(args)
+    for i, item in enumerate(args):
+        if not isinstance(item, list):
+            args[i] = [item] * len(urls)
+    args = list(zip(*args))
+
```

Comment:
```suggestion
assert len(args) == len(urls)
```

If its passed something like:
`download_url(["www.google.com", "www.yahoo.com"], [1, ])`

It will download only 1 file and not both files, since when you zip len of args will be lower then urls